content/prepare/install-java.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Install Java and Tools
+weight: 18
+---
+
+Our Raspberry Pi has started for the first time and we are now ready to add some Java to it!
+
+## Add Missing Dependencies
+
+Let's start with some helper tools that will be useful later.
+
+```shell
+$ sudo apt install -y i2c-tools vim git java-common libxi6 libxrender1 libxtst6
+```
+
+## Install Java
+
+There are many ways you can install Java. The easiest way to make Java available for both normal use and as root user (`sudo`), is the following approach which downloads one of the many distributions that are available.
+
+```shell
+$ cd ~/Downloads
+$ wget https://cdn.azul.com/zulu/bin/zulu21.38.21-ca-jdk21.0.5-linux_arm64.deb
+$ sudo dpkg -i zulu21.38.21-ca-jdk21.0.5-linux_arm64.deb
+$ rm zulu21.38.21-ca-jdk21.0.5-linux_arm64.deb
+```
+
+{{% notice warning %}}
+This approach only works on a 64-bit system as you can see from the `arm64` in the file name. For more info about running Java on a 32-bit system, check [Java for ARMv6/7/8](/documentation/java-for-arm/).
+{{% /notice %}}
+
+### Test Java Installation
+
+Now you should be able to check the Java version with the following command:
+
+```shell
+$ java -version
+openjdk version "21.0.5" 2024-10-15 LTS
```

Comment:
Now we talk about Java 21.

Reply:
yes, want people to install the latest. I added a comment to make this clear.